content/en/getting-started/faq.md

```diff
@@ -188,3 +188,19 @@ $ dig api.localstack.cloud
 If the result has some other status than `status: NOERROR,` your machine cannot resolve this domain.
 
 Some corporate DNS servers might filter requests to certain domains. Contact your network administrator to safelist` localstack.cloud` domains.
+
+### What's LocalStack Pro's patching schedule for security patches and bug fixes?
+
+Emergency vulnerabilities are treated with the highest priority and fixed on the same business day. The cloud provider (AWS) manages and maintains most of the infrastructure. All infrastructure configuration is fully encoded as code scripts (Infrastructure-as-Code) and can quickly be recreated in an emergency.
+
+### Does LocalStack have a security program that scans the containers for security issues?
+
+We are performing regular vulnerability scan reports for all software assets, including code and dependency scanning in our Github repositories (using Dependabot) and Docker image scans (using Trivy/Aquasec/Snyk image security scanner). If any vulnerabilities/CVEs are detected, they are fixed and usually rolled out within one business week.
```

Comment:
Possible change:  Our software assets are regularly checked for vulnerabilities, such as code issues and outdated dependencies. We use Dependabot to scan our GitHub repositories and Snyk (and other security tools) to scan our Docker images.